subgraphs/compound-forks/src/mapping.ts

```diff
@@ -254,7 +254,8 @@ export function _handleMarketEntered(
 
 export function _handleMarketListed(
```

Comment:
editing this breaks the build for all other compound forks. let's leave this function alone, but just overwrite the `market.name` in the bastion mapping code

Reply:
I set an empty default marketNamePrefix argument for `_handleMarketListed`, so this change will not break other compound forks. But I see your point, and changed to modify market.name afterwards, even though a bit redundant.